src/Microsoft.VisualStudio.AppDesigner/PropPages/PropPageUserControlBase.vb

```diff
@@ -3337,7 +3341,19 @@ NextControl:
         ''' <returns></returns>
         ''' <remarks></remarks>
         Protected Function GetProjectPath() As String
-            Return CStr(GetCommonPropertyValueNative("FullPath")) 'CONSIDER: This won't work for all project types, e.g. ASP.NET when project path is a URL
```

Comment:
I have a need to do the same for my "Reference Path " Properties Page and i was doing this .

''' <summary>
        ''' Gets the project's path
        ''' First look in PropertyDescriptor of a common property if not found look in ObjectsPropertyDescriptorsArray
        ''' </summary>
        ''' <returns></returns>
        ''' <remarks></remarks>
        Protected Function GetProjectPath() As String
            Dim PropertyName = "FullPath"
            Dim propertyDescriptor = GetCommonPropertyDescriptor(PropertyName)

            If propertyDescriptor Is Nothing Then
                propertyDescriptor = GetPropertyDescriptor(PropertyName)
            End If

            Return CStr(GetCommonPropertyValueNative(propertyDescriptor))
        End Function